lib/model/budget.dart

```diff
@@ -135,4 +164,16 @@ class BudgetMethods extends SossoldiDatabase {
 
     return await db.delete(budgetTable, where: '${BudgetFields.id} = ?', whereArgs: [id]);
   }
+
+  Future<int> deleteByCategory(int id) async {
+    final db = await database;
+
+    return await db.delete(budgetTable, where: '${BudgetFields.idCategory} = ?', whereArgs: [id]);
+  }
+
+  //PER TEST
+  Future<int> deleteAll() async {
+    final db = await database;
+    return await db.rawDelete("DELETE FROM $budgetTable");
+  }
```

Comment:
Remember to remove this on the final submission, if is a method that you just need to simplify your work and it won't go in the final release